flake.nix

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: Apache-2.0
+
+{
+  description = "mlkem-c-embedded";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+
+    flake-parts = {
+      url = "github:hercules-ci/flake-parts";
+      inputs.nixpkgs-lib.follows = "nixpkgs";
+    };
+  };
+
+  outputs = inputs@{ flake-parts, nixpkgs, ... }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      imports = [ ];
+      systems = [ "x86_64-linux" "aarch64-linux" "aarch64-darwin" "x86_64-darwin" ];
+      perSystem = { pkgs, system, inputs', ... }:
+        let
+          core = with pkgs; [
+            # formatter & linters
+            astyle # 3.4.10
+
+            # build dependencies
+            gcc-arm-embedded-13 # arm-gnu-toolchain-13.2.rel1
+            openocd # 0.12.0
+            python311Packages.pyserial # 3.5
```

Comment:
Those are not needed for the AArch64 repo. Please remove.

Reply:
Done in https://github.com/pq-code-package/mlkem-c-aarch64/pull/24/commits/e55890ea93839417830ae6db2b1c7d8e1dc08716